src/screens/Booking/components/Baggage/Baggage.styles.ts

```diff
@@ -0,0 +1,20 @@
+import {StyleSheet} from 'react-native';
+
+export default StyleSheet.create({
+  container: {
+    marginLeft: 15,
+    marginRight: 15,
+    marginBottom: 15,
+    marginTop: 15,
```

Comment:
marginLeft: 15,
    marginRight: 15,
    marginBottom: 15,
    marginTop: 15,
should be replaced with `margin:15`